pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -88,7 +88,7 @@ func (k *KogitoAppSpec) IsGitURIEmpty() bool {
 	if k == nil {
 		return true
 	}
-	if k.Build == nil || &k.Build.GitSource == nil {
+	if k.Build == nil || (len(k.Build.GitSource.ContextDir) == 0 && len(k.Build.GitSource.Reference) == 0 && len(k.Build.GitSource.URI) == 0) {
```

Comment:
sorry for the nitpick, but could you separate those checks on a new line?

Reply:
Sure, I'll update